source/docs/hardware/hardware-basics/robot-battery.rst

```diff
@@ -0,0 +1,188 @@
+
+Damaged batteries should be safely disposed of as soon as possible. All retailers that sell large SLA batteries, like car batteries, should be able to dispose of it for you. They may charge a small fee, or provide a small "core charge refund", depending on your state law.
+
+.. warning:: **DO NOT** attempt to "repair" damaged or non-functional batteries.
+
+Battery Construction & Tools
+-----------------------------
+
+Battery Leads
+^^^^^^^^^^^^^^^^^^^
+
+Standard 6AWG, SB50 battery leads normally come in the Kit of Parts, courtesy of AndyMark. Teams are highly encouraged to use these leads.
+
+//TODO cite rules here
+The leads must be copper, minimum size 6AWG and maximum length 12" - there is no maximum size. Teams are recommended to use the standard 6AWG, it is sufficient for almost all robots.
+The Anderson SB Connector may be either a standard PINK or RED SB50, or another Anderson SB connector such as the SB120. Teams are *STRONGLY* recommended to use the standard RED or PINK SB50 for interoperability, as the other color housings will not intermate and you will be unable to borrow batteries, chargers, or other equipment.
+Terminals for the battery tabs are available online and through electrical supply houses, sold by the accepted wire size and the tab hole diameter, sometimes listed on catalogs as a "post diameter".
+
+Teams are strongly recommended to use either use properly rated tools and stringent quality control practices for the crimping process, or use vendor-made leads. //TODO do we talk about crimping anywhere that this should link to?
+
+
+Battery Connections
+^^^^^^^^^^^^^^^^^^^^
+
+//TODO picture of fully tightened terminal
+
+The terminals must not move after they are torqued.
+It is a good idea to secure the lugs with either a #10 star washer or a nordlock washer system, in addition to a nylon locking nut.
+
+Do not use the split ring lock washer, they do not increase joint strength.
+
+..warning:: The lug and tab must be in direct contact, copper to copper: do not put a washer of any kind separating them.
+
+This connection must also be wrapped for electrical safety; electrical tape will work, but heatshrink that fits over the entire connection is recommended.
+
+.. image:: images/robot-battery/heatshrink.png
+  :alt: One terminal of an FRC battery fully covered in heatshrink.
+
+
+Battery Chargers
+^^^^^^^^^^^^^^^^^^^
+
+There are many good off the shelf battery chargers with 'maintenance modes' for 12V batteries. Teams are required to retrofit SB connectors onto chargers used at competitions. //TODO Cite rule
+
+Battery chargers may use no more than 6A charging current at competitions - most modern smart chargers for 12V SLA batteries do not use more than 6A, but this rating is generally shown on the box. //TODO cite rule
+
+Charging at lower amperages will avoid heat buildup, but take longer.
+
+Battery chargers must be retrofit to use SB50 connectors rather than stock clamp leads. //TODO cite rule
```

Comment:
You mention this twice.  I don't know of any rule that specifies this but I agree that it is highly recommended.

Reply:
R34 this year

Reply:
It was R41 in 2019 tho